bubblewrap.c

```diff
@@ -602,16 +670,13 @@ switch_to_user_with_privs (void)
 }
 
 static void
-drop_privs (void)
+drop_privs (bool keep_requested_caps)
 {
-  if (!is_privileged)
```

Comment:
This is also what's causing https://github.com/projectatomic/rpm-ostree/issues/855; we're now dropping `CAP_DAC_OVERRIDE`, so when rpm-ostree runs `%pre` scripts that do e.g. `groupadd`, they get `EPERM` on `/etc/gshadow`.